pkg/filehandler/filehandler.go

```diff
@@ -77,11 +87,13 @@ func JoinDirAndPath(dirName string, fileName string) string {
 func CreateEmptyJsonFileIfNotExists(fileName string) {
 	_, err := os.Stat(fileName)
 
-	if err != nil {
-		if os.IsNotExist(err) {
-			ioutil.WriteFile(fileName, []byte("{}"), 400)
+	if errors.Is(err, os.ErrNotExist) {
```

Comment:
In case the error is different that `ErrNotExist`, I think it should still be logged (think for example a permissions or quota issue), although not with fatal probably

Reply:
If it doesn’t exist and can’t be created I need to exit otherwise the registry won’t be created and we can have duplicity the next time we run it.